Header header
float64 airspeed
int32 casualties
int32 unavailability
int32 digi_val
